bin/jens-gc

```diff
@@ -84,21 +86,21 @@ def main():
         return 2
 
     try:
-        validate_directories(settings)
+        validate_directories(
```

Comment:
‎Indeed. But there should have been tests for it ;P                                                                                                                                                                                                                From: Nacho BarrientosSent: Friday, July 22, 2016 11:50To: cernops/jensReply To: cernops/jensCc: Dmytro Petruk; AuthorSubject: Re: [cernops/jens] Refactoring (#15)In bin/jens-gc:
> @@ -84,21 +86,21 @@ def main():
>          return 2
>  
>      try:
> -        validate_directories(settings)
> +        validate_directories(

Missing )?

—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or mute the thread.